Sources/MosaiqueAssetsPicker/Views/AssetDetailCell.swift

```diff
@@ -122,12 +127,18 @@ final class AssetDetailCell: UICollectionViewCell, AssetDetailCellBindable {
     }
 
     func setDownloading(_ isDownloading: Bool) {
-        if isDownloading, spinner == nil {
-            let spinner = UIActivityIndicatorView(style: .whiteLarge)
-            contentView.addSubview(spinner)
-            spinner.center = contentView.center
-            spinner.startAnimating()
-            self.spinner = spinner
+        if isDownloading {
+            let _spinner: UIActivityIndicatorView
+            if let c = self.spinner {
+                _spinner = c
+            } else {
+                _spinner = UIActivityIndicatorView(style: .whiteLarge)
+                self.spinner = _spinner
+            }
+            contentView.addSubview(_spinner)
```

Comment:
I didn't knew it was ok to use `addSubview` to a view already in a view hierarchy !

Reply:
@ntnmrndn 
oh, sorry, it's my mistake.
Actually, we can't say this is a good way. I think it's better to check the view already added before addSubview.
UIKit works fine if we call addSubview multiple times to the same view.